appengine/flexible/sparkpost/README.md

```diff
@@ -0,0 +1,13 @@
+# Python + Flask + SparkPost Example on Google App Engine Flexible
+
+> [SparkPost][sparkpost]—Modern email delivery services built for developers, by developers.
```

Comment:
`[][]` is a document reference but I have missed the `[sparkpost]: https://www.sparkpost.com/` definition at the bottom of the page. Fixing...